diff --git a/Loop/Models/GlucoseBasedApplicationFactorStrategy.swift b/Loop/Models/GlucoseBasedApplicationFactorStrategy.swift
index 41caa3d7..beaa4c4d 100644
--- a/Loop/Models/GlucoseBasedApplicationFactorStrategy.swift
+++ b/Loop/Models/GlucoseBasedApplicationFactorStrategy.swift
@@ -12,8 +12,8 @@ import LoopKit
 import LoopCore
 
 struct GlucoseBasedApplicationFactorStrategy: ApplicationFactorStrategy {
-    static let minPartialApplicationFactor = 0.20 // min fraction of correction when glucose > minGlucoseSlidingScale
-    static let maxPartialApplicationFactor = 0.80 // max fraction of correction when glucose > maxGlucoseSlidingScale
+    static let minPartialApplicationFactor = 0.50 // min fraction of correction when glucose > minGlucoseSlidingScale
+    static let maxPartialApplicationFactor = 0.90 // max fraction of correction when glucose > maxGlucoseSlidingScale
     // set minGlucoseSlidingScale based on user setting for correction range
     // use mg/dL for calculations
     static let minGlucoseDeltaSlidingScale = 10.0 // mg/dL
@@ -30,7 +30,7 @@ struct GlucoseBasedApplicationFactorStrategy: ApplicationFactorStrategy {
         let lowerBoundTarget = correctionRange.lowerBound.doubleValue(for: .milligramsPerDeciliter)
 
         // Calculate minimum glucose sliding scale and scaling fraction
-        let minGlucoseSlidingScale = GlucoseBasedApplicationFactorStrategy.minGlucoseDeltaSlidingScale + lowerBoundTarget
+        let minGlucoseSlidingScale = 70.0 // GlucoseBasedApplicationFactorStrategy.minGlucoseDeltaSlidingScale + lowerBoundTarget
         let scalingFraction = (GlucoseBasedApplicationFactorStrategy.maxPartialApplicationFactor - GlucoseBasedApplicationFactorStrategy.minPartialApplicationFactor) / (GlucoseBasedApplicationFactorStrategy.maxGlucoseSlidingScale - minGlucoseSlidingScale)
         let scalingGlucose = max(currentGlucose - minGlucoseSlidingScale, 0.0)
 
diff --git a/Loop/Models/LoopConstants.swift b/Loop/Models/LoopConstants.swift
index fb69c827..f293c769 100644
--- a/Loop/Models/LoopConstants.swift
+++ b/Loop/Models/LoopConstants.swift
@@ -50,7 +50,7 @@ enum LoopConstants {
     static let retrospectiveCorrectionEnabled = true
     
     // Percentage of recommended dose to apply as bolus when using automatic bolus dosing strategy
-    static let bolusPartialApplicationFactor = 0.4
+    static let bolusPartialApplicationFactor = 0.8
 
     /// Loop completion aging category limits
     static let completionFreshLimit = TimeInterval(minutes: 6)
