diff --git a/Loop/Models/LoopConstants.swift b/Loop/Models/LoopConstants.swift
index fb69c827..f293c769 100644
--- a/Loop/Models/LoopConstants.swift
+++ b/Loop/Models/LoopConstants.swift
@@ -50,7 +50,7 @@ enum LoopConstants {
     static let retrospectiveCorrectionEnabled = true
     
     // Percentage of recommended dose to apply as bolus when using automatic bolus dosing strategy
-    static let bolusPartialApplicationFactor = 0.4
+    static let bolusPartialApplicationFactor = 0.8
 
     /// Loop completion aging category limits
     static let completionFreshLimit = TimeInterval(minutes: 6)
